server/utils/cookieless_utils.ts

```diff
@@ -0,0 +1,188 @@
+/*
+
+ MIT License
+
+ Copyright (c) 2019 Looker Data Sciences, Inc.
+
+ Permission is hereby granted, free of charge, to any person obtaining a copy
+ of this software and associated documentation files (the "Software"), to deal
+ in the Software without restriction, including without limitation the rights
+ to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ copies of the Software, and to permit persons to whom the Software is
+ furnished to do so, subject to the following conditions:
+
+ The above copyright notice and this permission notice shall be included in all
+ copies or substantial portions of the Software.
+
+ THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ SOFTWARE.
+
+ */
+
+import { Looker40SDK } from '@looker/sdk'
+import { NodeSession } from '@looker/sdk-node'
+import type { IApiSection } from '@looker/sdk-rtl'
+import { DefaultSettings } from '@looker/sdk-rtl'
+import type { ApplicationConfig, LookerEmbedUser } from '../types'
+
+/**
+ * The functions in this file demonstrate how to call server
+ * side functions for cookieless embed. Please note that the
+ * implementation is exceedingly simplistic and only works for
+ * a single user in a development environment.
+ */
+
+// Application configuration
+let config: ApplicationConfig
+
+// Set the configuration
+export const setConfig = (conf: ApplicationConfig) => (config = conf)
+
+// The Looker session
+let lookerSession: NodeSession
+
+/**
+ * Create a Looker session using the Looker SDK. This session can be
+ * be shared amongst all embed users. If data is required for the embed
+ * user, this session can be used to create a session for the embed user,
+ * something like the following but not required for this demo.
+ * <pre>
```

Comment:
not important but I think most doc systems support tripletick with syntax highlighting? Don't know if this doc generator does, though.

Reply:
Changed to use back ticks and re-ran doc generation. No change to output so it seems that it does